.circleci/config.yml

```diff
@@ -403,6 +415,28 @@ jobs:
           coverage-reports: /tmp/workspace/ui/coverage/lcov.info
           project-token: $CODACY_PROJECT_TOKEN
 
+  api-contract-upload:
+    executor: pact-executor
+    steps:
+    - attach_workspace:
+        at: /tmp/workspace
+    - run:
+        name: Pact Broker Publish
+        working_directory: ~
+        command: |
+          pact-broker publish /tmp/workspace/pacts --broker-base-url "https://pact-broker.dsp-eng-tools.broadinstitute.org/" --consumer-app-version ${CIRCLE_SHA1} --branch ${CIRCLE_BRANCH}
+
+  api-contract-can-i-deploy:
+    executor: pact-executor
+    steps:
+    - attach_workspace:
+        at: /tmp/workspace
+    - run:
+        name: Pact Broker, can I deploy?
+        working_directory: ~
+        command: |
+          pact-broker can-i-deploy /tmp/workspace/pacts --broker-base-url "https://pact-broker.dsp-eng-tools.broadinstitute.org/" --pacticipant aou-rwb-api --version ${CIRCLE_SHA1}
```

Comment:
This checks to see whether we can deploy based on the results of the contract uploaded in `api-contract-upload`.